hugr-model/src/v0/binary/read.rs

```diff
@@ -274,9 +277,13 @@ fn read_term<'a>(bump: &'a Bump, reader: hugr_capnp::term::Reader) -> ReadResult
             model::Term::ApplyFull { symbol, args }
         }
 
-        Which::Quote(r#type) => model::Term::Quote {
-            r#type: model::TermId(r#type),
-        },
+        Which::Const(reader) => {
+            let reader = reader?;
+            model::Term::Const {
+                r#type: model::TermId(reader.get_type()),
```

Comment:
I'm surprised this isn't being covered by the float test...?

Reply:
This is the *term* for the type of constants. The float test has a const *node*. The const node has a parameter `(@ prelude.const-json (@ arithmetic.float.types.float64) "{\"c\":\"ConstF64\",\"v\":{\"value\":1.0}}" (ext))` which is of type `(const (@ arithmetic.float.types.float64) (ext))`, but that type does not occur itself in the example.